### DISCUSSION TITLE
Schnellaktionen auf eine Seite

### DISCUSSION THREAD

**Discussion by ghost** [osmandapp/OsmAnd#20575] [Ideas]
Aus meiner Sicht spricht nichts dagegen, die Interaktionen der Schnellaktionen zu komprimieren. Aktuell werden immer zwei Zeilen angezeigt und darunter die Karte.
Es macht aber nichts, wenn alle Schaltflächen auf einer Seite dargestellt werden und dabei die Karte überdeckt wird.
So spart man sich das Hin und herblättern auf den Schnellaktionsseiten.

**Comment by ghost** [comment]
<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/ce6e1629-8c02-47f7-993d-e0a80b360a74"  width="300" height="auto"/> </td>
  </tr>
</table>



Ich habe drei Seiten, die könnte man ohne Qualitätsverlust auch auf eine Seite bringen. Um eine Schnellaktion anzuklicken, muss man nicht die Karte sehen.

**Reply by DmitryAlexei** [reply]
You can try to create additional Quick Actions buttons or even single action Quick actions. For more information see https://osmand.net/docs/user/widgets/quick-action#extra-quick-action-widgets

And please submit issues in English to be understood by the majority of the GitHub users.

**Reply by DmitryAlexei** [reply]
You may also need to update your app to the latest version from Google Play

**Comment by ghost** [comment]
1. Ich habe die Schaltflächen teilweise mit bis zu 4 Merkmalen hinterlegt.
2. Wenn ich auf Englisch schreiben soll, bin ich raus. Ich muss mir die englischen Texte auch übersetzen lassen. Wo ist das Problem?
